constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/ConstraintSetParser.kt

```diff
@@ -905,8 +905,6 @@ fun parseWidget(
 ) {
     val reference = state.constraints(elementName)
     val constraints = element.names() ?: return
-    reference.width = Dimension.Wrap()
-    reference.height = Dimension.Wrap()
```

Comment:
are you sure ? this sets the width/height default to wrap, else you'd need to always have width/height specified no?

Reply:
Yes, it's already specified in ConstraintReference, but I suppose there's harm done in double checking.

Reply:
Added check in latest commit.